cogent/lib/gum/kernel/linux/errno.cogent

```diff
@@ -8,7 +8,7 @@
 -- @TAG(NICTA_GPL)
 --
 
```

Comment:
The errors numbers are spefic to the linux kernel. If there are errors specific to fs, I will refactor them.
And yes, I've a few more patches that share ADT's and more code between the implementations.